xbmc/addons/AddonDll.cpp

```diff
@@ -400,31 +390,5 @@ ADDON_STATUS CAddonDll::TransferSettings()
   return ADDON_STATUS_OK;
 }
 
-bool CAddonDll::CheckAPIVersion(int type)
-{
-  /* check the API version */
-  const char* kodiVersion = kodi::addon::GetTypeVersion(type);
-  const char* addonVersion = m_pDll->GetAddonTypeVersion(type);
```

Comment:
Shouldn't we remove GetAddonTypeVersion from Addon API, now that we do no longer check that data?